packages/types/src/chain/parachain/Parachain.ts

```diff
@@ -82,4 +82,8 @@ export class Parachain extends Chain {
   getAssetMin(asset: Asset): number {
     return this.assetsData.get(asset.key)?.min ?? 0;
   }
+
+  isRelay(): boolean {
```

Comment:
We could potentially just add a flag `isRelay` to `Parachain` class. Which by default would be false.